agilex_piper/piper.xml

```diff
@@ -0,0 +1,300 @@
+<mujoco model="piper_description">
+  <compiler angle="radian" meshdir="assets/" texturedir="assets/" />
+
+  <default>
```

Comment:
Omar, can you create a piper child class under the main default class and have all the other ones nested under it? Then make the root body have `childclass="piper"` and add `class='piper'` to the actuators.